content/docs/setup/kubernetes/upgrading-istio/index.md

```diff
@@ -255,3 +255,66 @@ spec:
     tls:
       mode: DISABLE
 {{< /text >}}
+
+## Migrating the `RbacConfig` to `ClusterRbacConfig`
+
+The `RbacConfig` is deprecated due to a [bug](https://github.com/istio/istio/issues/8825). You should
+migrate to `ClusterRbacConfig` if you are currently using `RbacConfig`. The bug reduces the scope of
+the object to be namespace-scoped in some cases. The `ClusterRbacConfig` follows the exact same
+specification as the `RbacConfig` but with correct cluster scope implementation.
+
+To migrate the `RbacConfig` to `ClusterRbacConfig`:
+
+1. We provide the `convert_RbacConfig_to_ClusterRbacConfig.sh` script to automate the migration,
+   depending on if you have the [Istio installation package](/docs/setup/kubernetes/download-release):
+
+    - If you have the Istio installation package
+
+        Run the script with the following command under the directory of Istio installation package:
+
+        {{< text bash >}}
+        $ ./tools/convert_RbacConfig_to_ClusterRbacConfig.sh
+        {{< /text >}}
+
+    - If you don't have the Istio installation package
+
+        Download and run the script directly with the following command:
+
+        {{< text bash >}}
+        $ curl -L https://raw.githubusercontent.com/istio/istio/master/tools/convert_RbacConfig_to_ClusterRbacConfig.sh | sh -
+        {{< /text >}}
+
+The script automates the following operations:
+
+1. Create a `ClusterRbacConfig` with same specification as the existing `RbacConfig`. This
+   is because Kubernetes doesn't allow to change the kind of a custom resource after it's created.
+
+    For example, if you have the following `RbacConfig`:
+
+    {{< text yaml >}}
+    apiVersion: "rbac.istio.io/v1alpha1"
+    kind: RbacConfig
+    metadata:
+      name: default
+    spec:
+      mode: 'ON_WITH_INCLUSION'
+      inclusion:
+        namespaces: ["default"]
+    {{< /text >}}
+
+    The script creates the following `ClusterRbacConfig`:
+
+    {{< text yaml >}}
+    apiVersion: "rbac.istio.io/v1alpha1"
+    kind: ClusterRbacConfig
+    metadata:
+      name: default
+    spec:
+      mode: 'ON_WITH_INCLUSION'
+      inclusion:
+        namespaces: ["default"]
+    {{< /text >}}
+
+1. Apply the configuration and wait for a few seconds to let the configuration to take effect.
```

Comment:
Do not use the imperative here. This process is performed by the script, and thus the ordered list is descriptive rather than prescriptive. A simple present sentence does the trick, just use "The script" as the subject. For example:
1. The script applies the configuration and waits for a few seconds to let the configuration to take effect.